contents/data_vis_aog.md

```diff
@@ -0,0 +1,231 @@
+# Data Visualization with AlgebraOfGraphics.jl {#sec:aog}
+
+> **_NOTE:_**
+> Don't forget to check [`AlgebraOfGraphics.jl` documentation](https://aog.makie.org).
+
+`AlgebraOfGraphics.jl` is a higher-level plotting package that uses `Makie.jl` under the hood.
+It is geared towards data visualization workflows with support for `DataFrame`s.
+`AlgebraOfGraphics.jl` abstracts away some common patterns in data visualization using an interface that was based on the "grammar of graphics" approach that is used by R's `ggplot2` package.
+
+The **grammar of graphics** is a framework that follows a layered approach to construct visualizations in a structure manner.
+There are four main types of layers:
+
+- **data** layer
+- **mapping** layer
+- **visual transformation** layer
+- **statistical transformation** layer
+
+`AlgebraOfGraphics.jl` allows you to construct all of these types of layers with functions that returns a `Layer` object,
+in which all of the information necessary will be encoded.
+You can then perform **two operations on layers**:
+
+- **multiplication with `*`**: this fuses two or more layers into a _single_ layer
+- **addition with `+`**: this superimposes two or more layers into a vector of `Layer`s
+
+Finally, as the name suggests, `AlgebraOfGraphics.jl` is more than a grammar but also an algebra for `Layer` objects.
```

Comment:
Maybe at the same level where Grammar if mentioned it. So that is less sparse and creates confusion.

Reply:
Please check if [6df1648](https://github.com/JuliaDataScience/JuliaDataScience/pull/289/commits/6df164833c7e4554560c06bdb4988e77f96c9aa4) solves this.